src/test/java/gov/healthit/chpl/aqa/stepDefinitions/ExcelUtil.java

```diff
@@ -0,0 +1,78 @@
+package gov.healthit.chpl.aqa.stepDefinitions;
+
+import java.io.FileInputStream;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.poi.xssf.usermodel.XSSFCell;
+import org.apache.poi.xssf.usermodel.XSSFSheet;
+import org.apache.poi.xssf.usermodel.XSSFWorkbook;
+
+/**
+ * Class ExcelUtil Utility class.
+ */
+public final class ExcelUtil {
+    private static XSSFSheet excelWSheet;
+    private static XSSFWorkbook excelWBook;
+    private static XSSFCell cell;
+
+    /**
+     * String path points to the location of the ChplApi.xlsx, includes all the APIs based on the categories.
+     */
+    private static String path = System.getProperty("user.dir") + "\\src\\test\\resources\\ChplApi.xlsx";
```

Comment:
Should use the `File.separator` thing to do file locations, as it's platform agnostic